Zend/zend_exceptions.c

```diff
@@ -772,21 +773,40 @@ void zend_register_default_exception(void) /* {{{ */
 		}
 	} ZEND_HASH_FOREACH_END();
 
-	INIT_CLASS_ENTRY(ce, "ErrorException", error_exception_functions);
-	error_exception_ce = zend_register_internal_class_ex(&ce, default_exception_ce);
-	error_exception_ce->create_object = zend_error_exception_new;
-	zend_declare_property_long(error_exception_ce, "severity", sizeof("severity")-1, E_ERROR, ZEND_ACC_PROTECTED);
+	INIT_CLASS_ENTRY(ce, "Error", NULL);
```

Comment:
Im' not sure at all about this name.

This will break BC as many people may have an Error class already, and that's why it is named ErrorException.
I think this kind of break needs consensus